servicemon/builtin_plugins/mongo_writer.py

```diff
@@ -0,0 +1,43 @@
+import os
+import sys
+import json
+import requests
+
+from servicemon.plugin_support import AbstractResultWriter
+
+
+class MongoDBResultWriter(AbstractResultWriter, plugin_name='mongo_writer',
+                          description='Sends results to a remote MongoDB database.'):
+
+
+    def begin(self, args, outfile=None):
+
+        self._outURL = 'http://vmnavo01.ipac.caltech.edu/cgi-bin/NAVOMonitor/nph-submit'
+
+        self._logFile = '/tmp/servicemon_' + str(os.getpid()) + '.log'
+
+    def end(self):
+        pass
+
+
+    def one_result(self, stats):
+
+        jsonStr = '{\n'
+
+        fields = stats.columns()
+        self.ncols = len(fields)
+
+        row = stats.row_values()
+
+        jsonStr = json.dumps(row)
+
+        print(jsonStr)
```

Comment:
Is this needed?  It will either collect in the output file, ir get thrown away when the output file is disabled by default.